From 99417d28245a067676d084d4774dc4fd9118943b Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Fri, 9 Mar 2018 07:13:55 +0100
Subject: [PATCH 68/74] jni: Resolve FM_DEVICE_PATH R/O open flag

 * hardware/qcom/fm/jni/android_hardware_fm.cpp:668:47:
    error: 'open' has superfluous mode bits; missing O_CREAT?

Change-Id: Ib6d4b27999c827309d151a0b4363f2a5f22d6b60
---
 jni/android_hardware_fm.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/jni/android_hardware_fm.cpp b/jni/android_hardware_fm.cpp
index 3d33caf..39110ab 100644
--- a/jni/android_hardware_fm.cpp
+++ b/jni/android_hardware_fm.cpp
@@ -659,7 +659,7 @@ static jint android_hardware_fmradio_FmReceiverJNI_acquireFdNative
     if(radio_path == NULL){
         return FM_JNI_FAILURE;
     }
-    fd = open(radio_path, O_RDONLY, O_NONBLOCK);
+    fd = open(radio_path, O_RDONLY, 0);
     if(isCopy == JNI_TRUE){
         env->ReleaseStringUTFChars(path, radio_path);
     }
-- 
2.17.1

